lib/core/src/lib/viewer/components/viewer.component.html

```diff
@@ -224,7 +224,7 @@ <h2>{{ 'ADF_VIEWER.LOADING' | translate }}</h2>
 
                         <ng-container *ngSwitchCase="'image'">
                             <adf-img-viewer [urlFile]="urlFileContent"
-                                            [nameFile]="displayName"
+                                            [nameFile]="displayName ? displayName : fileName"
```

Comment:
just an improvement, can be `displayName || fileName`

Reply:
Improved